executor/aggfuncs/func_sum_decimal.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2018 PingCAP, Inc.
```

Comment:
We'd better name this file to `func_sum.go` to follow the same file naming style with other aggregate functions.

Reply:
Is it better to separate SUM(decimal) and SUM(float) to single file?

Reply:
I think it's better to follow the existing flavors. Not only aggregate functions, but also other components, for example, expression and executors, they all put the code with same functionality in the same source file. 😄

Reply:
ok,follow the existing flavors

Reply:
please rename this file to `func_sum.go`